zcash_client_sqlite/src/wallet.rs

```diff
@@ -1964,6 +1974,70 @@ pub(crate) fn store_transaction_to_be_sent<P: consensus::Parameters>(
         }
     }
 
+    // Add the transaction to the set to be queried for transaction status. This is only necessary
+    // at present for fully-transparent transactions, because any transaction with a shielded
```

Comment:
```suggestion
    // at present for fully transparent transactions, because any transaction with a shielded
```
The rule is that adverbs modifying an adjective do not require a hyphen.